sql-tests/testdata/drop_metric.sql

```diff
@@ -0,0 +1,124 @@
+
+\unset ECHO
+\set QUIET 1
+\i '/testdata/scripts/pgtap-1.2.0.sql'
+
+SELECT * FROM plan(49);
+CREATE EXTENSION promscale;
+
+--
+-- Moved from TestSQLDropMetricChunk
+--
+
+DO $$
+DECLARE
+ s1_series_id BIGINT;
+ s2_series_id BIGINT;
+ s3_series_id BIGINT;
+BEGIN
+ -- Avoid randomness in chunk interval size by setting explicitly.
+ PERFORM _prom_catalog.get_or_create_metric_table_name('test');
+ PERFORM public.set_chunk_time_interval('prom_data.test', interval '8 hour');
+
+ -- this series (s1) will be deleted along with it's label
+ SELECT f.series_id 
+  FROM _prom_catalog.get_or_create_series_id_for_kv_array('test', ARRAY['__name__', 'name1'], ARRAY['test', 'value1']) f
+  INTO STRICT s1_series_id;
+ SELECT f.series_id
+  FROM _prom_catalog.get_or_create_series_id_for_kv_array('test', ARRAY['__name__', 'name1'], ARRAY['test', 'value2']) f
+  INTO STRICT s2_series_id;
+ SELECT f.series_id
+  FROM _prom_catalog.get_or_create_series_id_for_kv_array('test', ARRAY['__name__', 'name1'], ARRAY['test', 'value3']) f
+  INTO STRICT s3_series_id;
+
+ INSERT INTO prom_data.test(time,value,series_id)
+  VALUES
+   (E'2009-11-10 23:59:59.999+00',0.1,s1_series_id), -- this will be dropped (notice the - 1)
```

Comment:
What is this `E'...'` syntax? I'm not familiar with it?

Reply:
It's a PostgreSQL syntactic extension to allow C-style escapes (Section 4.1.2.2 https://www.postgresql.org/docs/14/sql-syntax-lexical.html). A leftover of me copy-pasting from a GUI client that is unnecessary here. Will remove.